startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
Chaos' process is not replaced until the restart. Unless something is misunderstood and there is no long process.